initfiles/componentfiles/thor/run_thor

```diff
@@ -43,6 +42,13 @@ while [ 1 ]; do
     fi
     $deploydir/makethorgroup
     sort $instancedir/slaves | uniq > $instancedir/uslaves.start
+    diff $instancedir/uslaves.start <(${INSTALL_DIR}/sbin/configgen -env ${CONFIG_DIR}/${ENV_XML_FILE} -c ${THORNAME} -listall2 | awk 'BEGIN {FS=",";} /ThorSlaveProcess/ { print $3 }' | sort | uniq) &>/dev/null
+    if [[ $? -ne 0 ]]; then
+      echo "ERROR:  dfsgroup $groupName is not the same as the environment.xml"
+      echo "If this was intentional, please run 'updtdalienv ${CONFIG_DIR}/${ENV_XML_FILE} -f'"
+      echo "Stopping $groupName"
+      exit 1
+    fi
```

Comment:
I don't think this works, unless the environment uses IP's (or "."), if it uses host names, then it will always mismatch the group (which is always IP based) and fail to start afaics.